etc/nginx.conf.template

```diff
@@ -19,7 +19,7 @@ http {
   }
 
   server {
-    listen 8080 deferred;
+    listen %NGINX_HTTP_PORT% deferred;
```

Comment:
who is going to provide this variable?

Reply:
changes below should do that. ahahah forgot to add that commit.